tests/components/plugwise/test_config_flow.py

```diff
@@ -89,3 +106,17 @@ async def test_form_cannot_connect(hass, mock_smile):
 
     assert result2["type"] == "form"
     assert result2["errors"] == {"base": "cannot_connect"}
+
+
+async def test_show_zeroconf_form(hass, mock_smile) -> None:
+    """Test that the zeroconf confirmation form is served."""
+    flow = config_flow.ConfigFlow()
```

Comment:
Please start all tests with the core interfaces. Here use `hass.config_entries.flow.async_init`. See our scaffold script template for good examples.

Reply:
Copied this from Elgato, but since I'm running into a wall on this anway with https://github.com/home-assistant/core/pull/37229#issuecomment-687613945 I'll probably should rewrite the whole bunch of zeroconf. I was using scaffolding but started looking into existing integrations along the way.

Reply:
The hue config flow tests have quite extensive discovery flow tests.

Reply:
Verifying my steps here as well (see comment link to other PR as well), but how should I formally call the zeroconf? There are no applicable examples that I see in scripts/scaffold and elgato/wled/homekit seem to follow this process. Basically I'm unable to get around feeding the discovery_info (which works) but than inserting the password.

Reply:
Start the flow zeroconf step with `hass.config_entries.flow.async_init` and pass the discovery info as `data` parameter. Then continue the flow with `hass.config_entries.flow.async_configure`.

Reply:
Yeah, turned out I was looking in the wrong direction, a try connect already returned leaving me blank. That way the normal process (which I tried already $ times didn't work out). It should be solved, merging in this branch as well in a minute